### PR TITLE
W-13498888-(W-13498827)-proxy-settings-update

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

**Review by ms-tlu — APPROVED** [review]
LGTM 👍

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 1]
```diff
@@ -11,12 +11,36 @@ After downloading and installing RPA Builder, configure the connection with RPA
...
 See xref:rpa-manager::usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a Users] for instructions. If you are in doubt, contact your organization's RPA administrator.
 
+[[connect-with-anypoint]]
+== Connect RPA Builder with RPA Manager (Anypoint Platform)
+
+After opening RPA Builder for the first time, configure the connection with RPA Manager by using your Anypoint Platform credentials:
+
+. Provide your RPA Manager instance address.
++
+image:anypoint-rpa-org-log-in.png[The Anypoint Platform login screen asking to configure the RPA Manager address, 35%, 35%]
```
Comment (hannanelson) [inline rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:21]:
```suggestion
image:anypoint-rpa-org-log-in.png[The Anypoint Platform login screen prompt to configure the RPA Manager address, 35%, 35%]
```

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 2]
```diff
@@ -11,12 +11,36 @@ After downloading and installing RPA Builder, configure the connection with RPA
...
++
+*Proxy Settings* enable you to configure a specific proxy to use when stablishing the connection with Anypoint Platform.
++
+By default, *Use System Proxy Settings* is enabled so that RPA Builder uses the same proxy configuration your system uses to access internet.
+. Log in using your Anypoint Platform account. 
++
+image:anypoint-rpa-user-log-in.png[The Anypoint Platform login screen asking to configure the login credentials, 35%, 35%]
```
Comment (hannanelson) [inline rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:28]: Line 21

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 3]
```diff
@@ -11,12 +11,36 @@ After downloading and installing RPA Builder, configure the connection with RPA
...
+
+After you log in, RPA Builder remains connected using the specified Anypoint Platform credentials. 
+
+To connect to a different RPA Manager instance or use different log in credentials:
+
+. In the top menu bar, navigate to *Options > Account*.
+. Click *Sign out*.
+. Provide the new RPA Manager address and log in credentials.
```
Comment (hannanelson) [inline rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:36]:
```suggestion
. Provide the new RPA Manager address and login credentials. 
```

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 4]
```diff
@@ -11,12 +11,36 @@ After downloading and installing RPA Builder, configure the connection with RPA
...
+. In the top menu bar, navigate to *Options > Account*.
+. Click *Sign out*.
+. Provide the new RPA Manager address and log in credentials. 
+
 [[connect-with-rpa-manager]]
-== Connect RPA Builder with RPA Manager
+== Connect RPA Builder with RPA Manager (Legacy)
 
-After opening RPA Builder for the first time, configure the connection with RPA Manager:
+If your organization was not migrated to Anypoint Platform yet, provide your RPA Manager user credentials to stablish the connection after opening RPA Builder for the first time:
```
Comment (hannanelson) [inline rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:41]:
```suggestion
If your organization is not migrated to Anypoint Platform, provide your RPA Manager user credentials to establish the connection after opening RPA Builder for the first time:
```

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 5]
```diff
@@ -11,12 +11,36 @@ After downloading and installing RPA Builder, configure the connection with RPA
...
 
-. In the top menu bar, select *Options* > *Repository*.
+. In the top menu bar, select *Options* > *Repository*. (This menu opens automatically the first time you open RPA Builder)
```
Comment (hannanelson) [inline rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:43]: Do you need the parenthetical or is it obvious from the UI?

File: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc [hunk 6]
```diff
@@ -32,7 +56,9 @@ The RPA Manager user name that RPA Builder uses to log in.
 The RPA Manager password that RPA Builder uses to log in.
 ** *Proxy Settings*
 +
-Configure your proxy settings if you access the internet through a proxy.
+Enables you to configure a specific proxy to use when connecting with RPA Manager.
```
[the remark this answers is not in the document]
Comment (hannanelson) [reply rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc:59]: (Not sure if that changes the meaning)

**Review by hannanelson — APPROVED** [review]

**Pull request merged**